public/client.js

```diff
@@ -167,9 +162,9 @@ socket.on('Display Message', function(data) {
             break;
         }
     }
-    var dateTime = new Date();
-    var hours = dateTime.getHours().toString(10);
-    var mins = dateTime.getMinutes().toString(10);
+    const dateTime = new Date();
+    const hours = dateTime.getHours().toString(10);
+    const mins = dateTime.getMinutes().toString(10);
```

Comment:
You need to change them into variables not constants. This creates a bug.

Reply:
In this case i have checked and review and their was no issues found

Reply:
I think after changing `var` to `const` you can't send a message, because we are changing values of `hours` and `mins` in lines `169` and `172` respectively.

Reply:
```suggestion
    let hours = dateTime.getHours().toString(10);
    let mins = dateTime.getMinutes().toString(10);
```